src/sharepoint/provisioning/objecthandlers/objectlists/objectlists.ts

```diff
@@ -257,7 +257,7 @@ export class ObjectLists extends ObjectHandlerBase {
         });
     }
     private GetFieldXmlAttr(fieldXml: string, attr: string) {
-        return $(jQuery.parseXML(fieldXml)).find("Field").attr(attr);
+        return jQuery(jQuery.parseXML(fieldXml)).find("Field").attr(attr);
```

Comment:
This is just for consistency. From what I can see we reference ```jQuery``` everywhere else. I still belive we should reference neither $ or jQuery :)